examples/material-ui-remix-ts/app/mui/MuiMeta.tsx

```diff
@@ -0,0 +1,5 @@
+import theme from "./theme";
+
```

Comment:
Why is an extra component necessary and not just adding `<meta ... />` into the layout component?

Reply:
My main approach was to keep modifying the main files as minimum as possible. We will only leave `<MuiMeta/>` on the root.tsx (no layout page anymore) file and the logic will be handled inside that.
 
So, if we need to change the metadata, we only update this file and people using this example, can simply update their MuiMeta.tsx file without the headache of touching core files again.